jasonwalsh/terraform-aws-artifactory
# fix: Add Terraform to Docker image

### Description

Installs Terraform in the Docker image.

See [73](https://circleci.com/gh/jasonwalsh/terraform-aws-artifactory/73)

### Resources Affected

- `Dockerfile`

### Verification Instructions

See the automatically triggered build in [Quay](https://quay.io/repository/jasonwalsh/circleci-terraform?tab=builds).

Resolves: N/A
